R/bs4_book.R

```diff
@@ -515,6 +516,54 @@ template_link <- function(html, xpath, href) {
   }
 }
 
+tweak_metadata <- function(html, path) {
+  file <- basename(path)
+  if (file == "index.html") {
+    return(invisible())
+  } else {
+    # Fix og:url
+    og_url <- xml2::xml_find_first(html, '//meta[@property="og:url"]')
+    base_url <- xml2::xml_attr(og_url, "content")
+    if (!grepl("/$", base_url)) {
+      base_url <- paste0(base_url, "/")
+    }
+    xml2::xml_set_attr(og_url, "content", paste0(base_url, file))
+    # Fix description
+    og_description <- xml2::xml_find_first(html, '//meta[@property="og:description"]')
+
+    if (file == "table-of-contents.html") {
+      xml2::xml_set_attr(
+        og_description,
+        "content",
+        paste("Table of contents;", xml2::xml_attr(og_description, "content"))
```

Comment:
Code tweaked, is it ok so or should I remove the entire snippet and store it in the table of contents related issue for adding it later?

Reply:
I think this would be cleaner history if you remove this, stash the change and put it back in a branch to table the TOC page. Or paste the git diff in the TOC page issue, or attach a patch file so that it can be easily apply when the other issue is tackled. 

Even if I understand the why, it does not feel right to mix things like this - PR should stay on topic.